namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/06-certificate.yaml

```diff
@@ -23,4 +23,4 @@ spec:
     name: letsencrypt-production
     kind: ClusterIssuer
   dnsNames:
-    - contacts-dev.hmpps.service.justice.gov.uk
+    - contacts-ui-dev.hmpps.service.justice.gov.uk
```

Comment:
We don't need this one, Prabash. This would undo the change in the other PR and make the cert contain the name contacts-ui-dev.xx.xx.xx. when we want contacts-dev.xx.xx.   Just close this PR, and let the other go through, and it will be all good.  Thanks.